.github/actions/assign-on-ptal/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3-slim AS builder
```

Comment:
This looks very heavy, is there a way to use the native version?

Reply:
My comment applies especially since you're only using this to run a Python script.